practice/react-practice-one/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,46 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import components */
+import Button from '../Buttons/Button';
+
+/* Import CSS */
+import './pagination.css';
+
+const Pagination = ({ currentPage, totalRecords, pageLimit, onPageChange }) => {
+  const totalPages = Math.ceil(totalRecords / pageLimit);
+
+  const handleClick = (page) => {
+    if (page !== currentPage) {
+      onPageChange(page);
+    }
+  };
+
+  const renderButtons = () => {
+    const buttons = [];
+
+    for (let i = 1; i <= totalPages; i++) {
+      buttons.push(
+        <Button
+          key={i}
```

Comment:
@JennyDoJD Don't use index as key 
Refer https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66.
You should do more research on this

Reply:
I fixed it